python/tests/conftest.py

```diff
@@ -63,7 +63,7 @@ def tmp_path() -> Generator[str, None, None]:
     "spark.python.worker.reuse": "false",
     "spark.driver.host": "127.0.0.1",
     "spark.task.maxFailures": "1",
-    "spark.driver.memory": "5g",
+    "spark.driver.memory": "128g",
```

Comment:
Will this severely limit where this can run?

Reply:
Revised to 32g. The added test leveraged disk and finished in about 10 mins. Seems acceptable as the dataset itself is large. 

Tried setting this to 960g, while physical memory has only 128g. All tests in test_logistic_regression.py run fine. Seems a larger value did not limit the tests to run.